### DISCUSSION TITLE
Date Picker and languages

### DISCUSSION THREAD

**Discussion by 7576457** [flet-dev/flet#4317] [Q&A]
How can I adapt the ```ft.DatePicker``` control for different languages?

![изображение](https://github.com/user-attachments/assets/55b21357-b022-40ff-ae57-4ec901320e38)

I know that it's possible to change the "ok" or "cancel" labels, but how can I change the month names?

**Comment by FeodorFitsner** [comment]
You can change the locale for the entire page (app): https://flet.dev/docs/controls/page#locale_configuration
DatePicker will adjust to that locale.

**Reply by ndonkoHenri** [reply]
https://python.plainenglish.io/app-localization-in-flet-5b523e83ca89

In the prerelease (and upcoming release), you are/will be able to change just the locale of the date picker only through the theme:
```py
page.theme = ft.Theme(date_picker_theme=ft.DatePickerTheme(locale=ft.Locale(...)))
```
